### ISSUE TITLE
Failure to launch Assassin's Creed games BLUS30991 and BLUS31193

### ISSUE THREAD

**Issue by rpcstest8** [RPCS3/rpcs3#9397]
## Quick summary
Trying to launch BLUS30991 or BLUS31193, previously "ingame" titles, fails.  No window launches.

## Details
Possible regression?  Unable to find anyone else that has these titles to verify.

#### 1. Please attach RPCS3's log.
[BLUS30991.zip](https://github.com/RPCS3/rpcs3/files/5662962/BLUS30991.zip)
[BLUS31193.zip](https://github.com/RPCS3/rpcs3/files/5662963/BLUS31193.zip)

#### 3. Please attach screenshots of your problem.
* Enable performance overlay with at least medium level of detail.
You can find it in _Emulator_ tab in _Settings_.

#### 5. Please provide your system configuration:
* OS: Operating system: Windows, Major: 10, Minor: 0, Build: 19041, Service Pack: none, Compatibility mode: 0
* CPU: Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz | 6 Threads | 31.93 GiB RAM | TSC: 3.696GHz | AVX+ | FMA3 | TSX-FA
* GPU: GeForce RTX 2070 SUPER
* Driver version: 457.51.0.0
* etc.

#### Please include.
* Anything else you deem to be important

**Comment by AniLeo** [comment]
Missing build that regressed those

**Comment by rpcstest8** [comment]
Tested on RPCS3 0.0.13-11355-a7e5c255
Tested back as far as I could (0.0.5) but unable to test back to the last confirmed logs of ingame in the 0.0.4 range due to those installers not being hosted on github anymore.

**Comment by rpcstest8** [comment]
Found a solution to launch the games, but i'm not sure it's valid.  In essence, you need to go into the PS3_GAME\USRDIR directory and create a folder ASSASSIN_ISO.  I'm not sure what this implies for what the emulator is not correctly doing, but without doing that, neither title will launch.

**Comment by Darkhost1999** [comment]
From what I've been able to learn:
This issue is not the fault of RPCS3.
It may be of how the game is dumped and the method for how the game was transferred from console to PC.

In my current experience of using Multiman and an FTP server ASSASSIN_ISO was already included in the PS3GAME/USRDIR before anybody informed me about the possible solution.

After being informed of that possible solution I tested Multiman using an MBR FAT32 USB. After using PS3merge on PC ASSASSIN_ISO was just gone and therefore needed to be created.

Long story short I see this issue is resolved for the op and I don't believe this is an emulation issue as the ps3 console would have the same issues if failure to extract/dump all necessary files from the disc.

**Comment by MSuih** [comment]
Seems to have been resolved
